### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by evanSe**
### Context
This pull request addresses issues raised by a recent npm audit.
- remove `cpy-cli` npm package as it is no longer in use
- remove stylelint rules as per stylelint migration guide https://github.com/stylelint/stylelint/blob/main/docs/migration-guide/to-15.md
- add `--bundleConfigAsCjs` to rollup builds only forces the config to be loaded as a cjs

**Only 10 moderate issues remain** which boils down to
- [vue-jest](https://github.com/advisories/GHSA-72xf-g2v4-qvf3)
- [tough cookie issue dtslint](https://github.com/advisories/GHSA-72xf-g2v4-qvf3)
- [ssr forge dtslint](https://github.com/advisories/GHSA-p8p7-x288-28g6)

Those 2 dependencies with issues do not affect a packaged build

### How has this been tested?
locally

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Additional language file or change to the existing one (translations)

### Related issue(s):
1. https://github.com/handsontable/dev-handsontable/issues/1816

### Affected project(s):
- [x] `handsontable`
- [x] `@handsontable/angular`
- [x] `@handsontable/react`
- [x] `@handsontable/vue`
- [x] `@handsontable/vue3`

[skip changelog]

### CODE REVIEW

**Review by budnix — APPROVED** [review]
Looks good

File: wrappers/react/package.json [hunk 1]
```diff
@@ -92,11 +91,11 @@
     "handsontable": ">=14.0.0"
   },
   "scripts": {
-    "build": "npm run clean && npm run build:commonjs && npm run build:es && npm run build:umd && npm run build:min",
-    "build:commonjs": "cross-env NODE_ENV=cjs rollup -c",
-    "build:umd": "cross-env NODE_ENV=umd rollup -c",
-    "build:es": "cross-env NODE_ENV=es rollup -c",
-    "build:min": "cross-env NODE_ENV=min rollup -c",
+    "build": "npm run clean && npm run build:es && npm run build:umd && npm run build:min",
```
Comment (budnix) [inline wrappers/react/package.json:94]:
```suggestion
    "build": "npm run clean && npm run build:commonjs && npm run build:es && npm run build:umd && npm run build:min",
```
After the changes, running `npm run build` does not build the dist files for CommonJS.

**Review by budnix — CHANGES_REQUESTED** [review]

File: wrappers/react/package.json (hunk 1 above)
Comment (evanSe) [reply wrappers/react/package.json:94]: Thanks, accidentally removed during testing

**Review by budnix — DISMISSED** [review]
NPM Audit job fails. Is that ok?

**Review by budnix — APPROVED** [review]

**Pull request merged**